app/konnect/configure/runtime-manager/runtime-instances/upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade a Runtime to a New Version
+no_version: true
+---
+
+You can upgrade runtimes to a new {{site.base_gateway}} version by bringing
+up new runtime instances, then shutting down the old ones. This is the best
+method for high availability, as the new node starts processing data before the
+old one is removed. It is the cleanest and safest way to upgrade with no
+proxy downtime.
+
+1. Provision a new runtime instance through the Runtime Manager:
+  * [Docker](/konnect/runtime-manager/runtime-instances/gateway-runtime-docker)
+  * [Linux](/konnect/runtime-manager/runtime-instances/gateway-runtime-conf)
+  * [Kubernetes (Helm)](/konnect/runtime-manager/runtime-instances/gateway-runtime-kubernetes)
+
+2. Open the Runtime Manager, then choose a runtime group.
+Make sure that your new runtime instance appears in the list of runtime
+instances, is in _Connected_ status, and that it was last seen _Just Now_.
+For example:
+
+    ![Connected runtimes in Runtime Manager](/assets/images/docs/konnect/konnect-runtimes-connected.png)
+
+3. If the new runtime instance is connected and functioning, you can disconnect and
+shut down any existing older instances that you're replacing.
+
+    {:.note}
+    > You can't shut down runtime instances from within Runtime Manager.
```

Comment:
Should we also mention that old instances will show as connected for `X hours` after they're removed? There's been some confusion around this in the past

Reply:
I wasn't aware of this issue. Do you have any tickets or anything to point to that has the length of time?

Reply:
Went with this for now:
>  You can't shut down runtime instances from within Runtime Manager. Old
    instances will also remain listed as `Connected` in Runtime Manager for a
    few hours after they have been removed or shut down.